devU-api/src/containerAutoGrader/containerAutoGrader.model.ts

```diff
@@ -0,0 +1,46 @@
+import {
+    JoinColumn,
+    OneToOne,
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+} from 'typeorm'
+
+import AssignmentModel from '../assignment/assignment.model'
+
+@Entity('container_auto_grader')
+export default class ContainerAutoGraderModel {
+    @PrimaryGeneratedColumn()
+    id: number
+
+    @CreateDateColumn({ name: 'created_at' })
+    createdAt: Date
+
+    @UpdateDateColumn({ name: 'updated_at' })
+    updatedAt: Date
+
+    @DeleteDateColumn({ name: 'deleted_at' })
+    deletedAt?: Date
+
+    @Column({ name: 'assignment_id' })
+    @JoinColumn({ name: 'assignment_id' })
+    @OneToOne(() => AssignmentModel)
```

Comment:
This should be ManyToOne since we are allowing multiple graders per assignment

Reply:
updated via commit id[ca6a6fd]